pywinauto/controls/uia_controls.py

```diff
@@ -1017,3 +986,216 @@ def check_button(self, button_identifier, make_checked, exact=True):
 
         self.actions.logSectionEnd()
         return button
+
+
+# ====================================================================
+class TreeItemWrapper(uiawrapper.UIAWrapper):
+
+    """Wrap an UIA-compatible TreeItem control
+
+    In addition to the provided methods of the wrapper
+    additional inherited methods can be especially helpful:
+    * select()
+    * extend()
+    * collapse()
+    * is_extended()
+    * is_collapsed()
+    * click_input()
+    * rectangle()
+    and many others
+    """
+
+    _control_types = [
+        IUIA().UIA_dll.UIA_TreeItemControlTypeId
+    ]
+
+    # -----------------------------------------------------------
+    def __init__(self, elem):
+        """Initialize the control"""
+        super(TreeItemWrapper, self).__init__(elem)
+
+    # -----------------------------------------------------------
+    def is_checked(self):
+        """Return True if the TreeItem is checked
+
+        Only items supporting Toggle pattern should answer.
+        Raise NoPatternInterfaceError if the pattern is not supported
+        """
+        return (self.iface_toggle.ToggleState_On == toggle_state_on)
+
+    # -----------------------------------------------------------
+    def ensure_visible(self):
+        """Make sure that the TreeView item is visible"""
+        self.iface_scroll_item.ScrollIntoView()
+
+    # -----------------------------------------------------------
+    def get_child(self, child_spec, exact=False):
+        """Return the child item of this item
+
+        Accepts either a string or an index.
+        If a string is passed then it returns the child item
+        with the best match for the string."""
+
```

Comment:
Minor: triple double quotes should be on a separate line, first statement in the method should be on a line right after the doc string.